projects/phone/src/lib/masks/phone/processors/cut-phone-postprocessor.ts

```diff
@@ -0,0 +1,20 @@
+import {MaskitoPostprocessor} from '@maskito/core';
+import {CountryCode, MetadataJson} from 'libphonenumber-js/core';
+
+import {cutPhoneByValidLength} from '../utils';
+
+const MIN_LENGTH = 3;
+export function maskitoCutPhonePostprocessorGenerator(
```

Comment:
What do you think about this naming ?

```suggestion
export function maskitoPhoneLengthPostprocessorGenerator(
```

Reply:
\+ new file name `phone-length-postprocessor.ts`